system-setup.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python3
```

Comment:
We can't switch to python3 here, because there are parts of the automation environment (e.g. tests) that rely on python2. We do want to do that eventually, but it will take some effort.

Reply:
Right. The python version being deployed is about more than just the build. Thank you!